lib/go-atscfg/parentdotconfig.go

```diff
@@ -778,13 +778,13 @@ type dsesSortByName []DeliveryService
 func (s dsesSortByName) Len() int      { return len(s) }
 func (s dsesSortByName) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 func (s dsesSortByName) Less(i, j int) bool {
-	if s[i].XMLID == nil {
+	if &s[i].XMLID == nil || s[i].XMLID == "" {
```

Comment:
The `&s[i].XMLID == nil` check can be removed.

Reply:
removed